infinitest-runner/src/main/java/org/infinitest/config/FileBasedInfinitestConfigurationSource.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Infinitest, a Continuous Test Runner.
+ *
+ * Copyright (C) 2010-2013
```

Comment:
isn't a new class?
copyright should be set to 2016 ?
The list of author need to cleaned and have just your name?